su-template/default/layout/html/layout.erb

```diff
@@ -43,6 +43,8 @@
         <div class="clear"></div>
       </div>
 
+      <iframe id="search_frame" src="<%= @nav_url %>"></iframe>
+
       <div id="content"><%= yieldall %></div>
```

Comment:
Hmmm... why didn't I have it ? I sync'd before I made changes. (I thought.) Let me make sure I have all his changes from that commit, and PR the "layout.erb" file.
Thanks Jim!